Yosemite/Yosemite/Model/Storage/WooShippingPredefinedOption+ReadOnlyConvertible.swift

```diff
@@ -0,0 +1,22 @@
+import Foundation
+import Storage
+
+// Storage.WooShippingPredefinedOption: ReadOnlyConvertible Conformance.
+//
+extension Storage.WooShippingPredefinedOption: ReadOnlyConvertible {
+    /// Updates the Storage.WooShippingPredefinedOption with the a ReadOnly WooShippingPredefinedOption.
+    ///
+    public func update(with option: Yosemite.WooShippingPredefinedOption) {
+        self.title = option.title
+        self.providerID = option.providerID
+
```

Comment:
Same as above, shouldn't we also update the rest of the properties (`predefinedPackages`)?